nixos/lib/systemd-lib.nix

```diff
@@ -286,17 +291,11 @@ in rec {
     };
   };
 
-  serviceConfig = { name, config, ... }: {
+  mkServiceConfig = path: { name, config, ... }: {
     config = mkMerge
       [ { # Default path for systemd services.  Should be quite minimal.
```

Comment:
I assume this comment should be moved to the `serviceConfig =…` assignment below, as the list of packages isn't here anymore.

Reply:
Pushed to fc91cdb5bcd909f32496c4b69347fb1bd0755a77.